website/docs/d/public_key.html.md

```diff
@@ -38,14 +72,14 @@ The following attributes are exported:
   ["Authorized Keys"](https://www.ssh.com/academy/ssh/authorized_keys/openssh#format-of-the-authorized-keys-file)
   format. This is populated only if the configured private key is supported:
   this includes all `RSA` and `ED25519` keys, as well as `ECDSA` keys with curves
-  `P256`, `P384` and `P521`.
+  `P256`, `P384` and `P521` (see [above](#limitations-ecdsa-with-p224-elliptic-curve)).
```

Comment:
Nit: "here" like linking

```suggestion
  `P224` ([with limitations](#limitations-ecdsa-with-p224-elliptic-curve)), `P256`, `P384` and `P521`.
```

Reply:
Actually, there is an issue with this one: for a curve `P224` we actually DON'T populate that field.